doc/contribute.md

```diff
@@ -1,61 +1,19 @@
 # How to contribute?
 
-## Report bugs
-
-Have you found a bug in the firmware? [Create an issue on Github](https://github.com/InfiniTimeOrg/InfiniTime/issues) explaining the bug, how to reproduce it, the version of the firmware you use...
-
-## Write and improve documentation
-
-Documentation might be incomplete, or not clear enough, and it is always possible to improve it with better wording, pictures, photo, video,...
-
-As the documentation is part of the source code, you can submit your improvements to the documentation by submitting a pull request (see below).
-
-## Fix bugs, add functionalities and improve the code
-
-You want to fix a bug, add a cool new functionality or improve the code? See *How to submit a pull request below*.
-
-# How to submit a pull request?
-
-## TL;DR
-
- - Create a branch from develop
- - Work on a single subject in this branch. Create multiple branches/pulls-requests if you want to work on multiple subjects (bugs, features,...)
- - Test your modifications on the actual hardware
- - Check your code against the [coding conventions](/doc/coding-convention.md) and [clang-format](../.clang-format) and [clang-tidy](../.clang-tidy)
- - Clean your code and remove files that are not needed
- - Write documentation related to your new feature if applicable
- - Create a pull request and write a great description about it: what does your PR do, why, how,... Add pictures and video if possible
- - Wait for someone to review your PR and take part in the review process
- - Your PR will eventually be merged :)
+- [Report bugs](https://github.com/InfiniTimeOrg/InfiniTime/issues/new?assignees=&labels=bug&template=bug-report.yaml)
+- Write and improve documentation
+  - Documentation might be incomplete, or not clear enough, and it is always possible to improve it with better wording, pictures, videos,...
+  - As the documentation is part of the source code, you can submit changes to the documentation by creating a pull request (see below).
+- Fix bugs, add functionalities and improve the code
+  - You want to fix a bug, add a cool new functionality or improve the code? See *How to create a pull request* below.
+
+## How to create a pull request?
+
+- Fork the project, create a [feature branch](branches.md) from develop and give it a short name that explains the topic of your changes.
```

Comment:
not all sentences have a trailing `.`. I think we should add a `.` or remove them for all

for me I'd remove the trailing `.` altogether